indigo/docs/guides/howto-hello-indigo-tyrian.md

```diff
@@ -0,0 +1,418 @@
+---
+id: howto-responsive-ui-with-tyrian
+title: How to make a custom entity
+---
+
+## Responsive UI Using Tyrian
+
+There tend to be two primary types of UI in games: In-game UI (such as health over a
+player), and overlay (such as a HUD or dialog box). In-game UI tends not to
+worry too much about the screen size, and is dealt with primarily by the
+magnification configuration within Indigo. But what about overlays and HUD's?
+These tend to scale with the size of the screen and the position of such
+elements may change depending on the capabilities of the display. Many game
+engines provide a way to create overlay UI elements that scale in some way to
+the space they're in using a separate UI system. In Indigo we're exporting to
+JavaScript so it makes sense for us to use other web technologies for a problem
+that is already, ultimately, solved.
+
+Using Tyrian we can utilise HTML and CSS to ensure our overlay UI scales
+no matter what screen size we use.
+
+In this guide we'll take the basic `hello-indigo` example and add a button and
+a counter that scales with the size of the screen. It's a pretty basic example to
+be honest, but it should give you an idea of what can be achieved.
+
+## Setup
+
+### Setting up the Environment
+
+For this guide we'll be using the `hello-indigo` example in either
+[mill](https://github.com/PurpleKingdomGames/hello-indigo) or
+[sbt](https://github.com/PurpleKingdomGames/hello-indigo-sbt).
+In the existing implementation clicking on the dots will create a new dot
+that then rotates around the canvas. We'll modify this so that clicking a button
+does this job instead.
+
+You'll want to upgrade `hello-indigo` to use `IndigoGame`. To do this either
+follow [this guide]()
+or replace `HelloIndigo.scala` with
+[this](https://gist.github.com/hobnob/c24f00936e91a7b7e5d644d19e4f1b32)
```

Comment:
Is it worth these gist's you're fond of living on the pkg account? (if that's a thing?) I'm not concerned, just asking.

Reply:
Unfortunately we can't have them on the PKG account 🙁  Worth keeping in mind though, just in case they add it